### DISCUSSION TITLE
Is it possibile to create lookup fields where the data is provided by external services?

### DISCUSSION THREAD

**Discussion by GianlucaLostuzzo** [Budibase/budibase#6822] [Help] [answered]
Hi everyone.
Is it possible to create lookup columns in the database provided by the tool where the selectable data is retrieved from external services?
For example: I’ve created a region column where I want that the selectable regions are retrieved by a government service that gives a list of all the regions of a country (e.g. Italy).

Thanks in advance for your help!

**Comment by melohagan** [comment, marked as answer]
Hi @GianlucaLostuzzo 

I'm not quite sure what you're looking to do here, but you may be able to use an Options Picker: https://docs.budibase.com/docs/option-picker

And then use the value of the options picker as a binding for a REST query: https://docs.budibase.com/docs/rest-bindings

**Reply by GianlucaLostuzzo** [reply]
Thanks @melohagan, your suggestion helped me solve the problem!
